docs/nextcloud.md

```diff
@@ -430,3 +434,127 @@ HTTP/1.1 204 No Content
 - 400 Bad Request, when the account is not configured for NextCloud
 - 401 Unauthorized, when authentication to the NextCloud fails
 - 404 Not Found, when the account is not found or the file is not found on the Cozy
+
+## GET /remote/nextcloud/:account/trash/*
+
+This route can be used to list the files and directories inside the trashbin
+of NextCloud.
+
+### Request (list)
+
+```http
+GET /remote/nextcloud/4ab2155707bb6613a8b9463daf00381b/trash/ HTTP/1.1
+Host: cozy.example.net
+Authorization: Bearer eyJhbG...
+```
+
+### Response (list)
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+```
+
+```json
+{
+  "data": [
+    {
+      "type": "io.cozy.remote.nextcloud.files",
+      "id": "613281",
+      "attributes": {
+        "type": "directory",
+        "name": "Old",
+        "path": "/trash/Old.d93571568",
```

Comment:
@cballevre be careful, the path for things inside the trash is a bit weird and cannot be computed with just the parent and the name, so it was added to the response

Reply:
Interesting clarification. I saw that you added the `path` everywhere. In cozy-client, I add a second attribute named `parentPath`. It corresponds to the folder path. This is to make it easier to replay requests with sift.js ([commit with some explanation](https://github.com/cozy/cozy-client/pull/1488/commits/05293215bef289f0170b617bc278057916b5e778)). Wouldn't it make sense to add it to the back as well, so we wouldn't have to calculate anything on the front?